tests/handlers/test_agent.py

```diff
@@ -30,3 +41,45 @@ def test_post_agent_unauthenticated(test_client, db_session):
 
     assert response.status_code == 403
     assert response.json()['detail'] == "Permission denied."
+
+
+@pytest.mark.asyncio
+async def test_close_websocket_connect(
```

Comment:
This test should basically go as follows:
1. create an agent
2. connect websocket using the agent credentials
3. call cleanup through `test_client`
4. assert everything went well